### PR TITLE
提交一篇blog~

### PR DESCRIPTION
**Pull request by rixingyike**
Please make sure to check the following boxes creating your pull request, thank you very much!

- [ ] Add Blog Header, including title, author, image, categories, tags, etc.  

    For example:  

    ```md
    ---
    title: Your Great Blog Title
    author: your_github_id
    categories: project
    tags:
      - python
      - padlocal
    image: /assets/2021/02-your-blog-slug/teaser.jpg
    ---

    One line bio

    Your blog post content Your blog post content
    Your blog post content Your blog post content
    Your blog post content Your blog post content
    ```

    > It will be easier to start drafting your post by using a existing post file as your template, for example, [this one](https://github.com/wechaty/wechaty.js.org/blob/master/jekyll/_posts/2021-01-14-wechaty-puppet-service.md)

- [ ] Keep all filenames lowercase, and use `-` to connect words instead of space. e.g. `2017-10-06-wechat-pc-impactor.md` instead of `2017-10-06-WeChat PC Impactor.md`
- [ ] Embed video from YouTube (if there's any)
- [ ] Save photo to local before publishing, to a folder named by `${MONTH}-${YOUR_BLOG_SLUG}/` under `/assets/2021/` directory.
- [ ] Select a beautiful and meaningful teaser image for your blog
- [ ] Create your contributor profile (if you are a first time contributor)

    For excample:

    > jekyll/_contributors/lijiarui.md

    ```yaml
    ---
    name: Jiarui LI (李佳芮)
    site: https://lijiarui.github.io
    bio: 句子互动创始人 & CEO，微软人工智能最具价值专家 (AI MVP)
    avatar: /jekyll/assets/contributors/lijiarui/avatar.png
    email: rui@juzi.bot
    twitter: https://weibo.com/u/2175505900
    ---
    ```

  Here is the example file:

  - lijiarui.md: <https://github.com/wechaty/wechaty.js.org/blob/master/jekyll/_contributors/lijiarui.md>
  - It will shown like this on wechaty website: <https://wechaty.js.org/contributors/lijiarui/>

To learn more about the contribute guideline, visit: <https://wechaty.js.org/docs/contributor-program/>

### CODE REVIEW

File: jekyll/_posts/2021-02-09-how-to-use-wechaty-to-ask-for-payment-before-user-joining-the-group.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+当然了这个版本还存在一些问题，例如机器人助手依据昵称管理员权限，这存在漏洞。Wechaty中Contact对象有一个alias方法，可以设置/获取联系人备注，可以使用这个方法代替name检验管理员权限。
+
+后续作者可能对这个项目进行不断完善，如果你有什么建议，欢迎提出来，也欢迎提交PR。
+
+## 使用视频
+
+作者为这个版本的使用录了一个视频：https://mp.weixin.qq.com/s/TUKmK7IgJElECt7hNq5QEA
+
+这里还有一个Youbute版本：https://youtu.be/Rujwzt0B9K8
```
Comment (huan) [inline jekyll/_posts/2021-02-09-how-to-use-wechaty-to-ask-for-payment-before-user-joining-the-group.md]: I'd like to suggest that we can add an embed version for the YouTube video by adding the following code:

```md
{% include iframe.html src="https://youtu.be/Rujwzt0B9K8" %}
```

See: https://wechaty.js.org/2020/08/24/add-video-to-wechaty-blog/

**Review by huan — CHANGES_REQUESTED** [review]
Thanks for sharing your great article & video!

Please make the CI green so that we can merge this PR, and feel free to let me know if you have any questions about the lining rules.

**Review by rixingyike — COMMENTED** [review]
修改了油管视频格式~

**Comment by rixingyike** [comment]
done~~~

**Comment by rixingyike** [comment]
《使用小微商户 + Wechaty构建一个可付费的私域流量运营助手》

**Comment by rixingyike** [comment]
> Thanks for sharing your great article & video!
> 
> Please make the CI green so that we can merge this PR, and feel free to let me know if you have any questions about the lining rules.

done~

**Review by huan — APPROVED** [review]
LGTM

**Comment by huan** [comment]
Your post looks great, merged!

**Pull request merged**